core/docker/compose.sh

```diff
@@ -21,6 +21,14 @@ if [ "$1" == "init" ]; then
     docker cp "$composeFilePath"/importer/volume/mysql.cnf hapi-mysql-helper:/conf.d/mysql.cnf
     docker rm hapi-mysql-helper
 
+    PASSWORD_SALT=$(openssl rand -hex 16)
+    PASSWORD_HASH=$(echo -n $PASSWORD_SALT${OPENHIM_ROOT_PASSWORD-instant101} | openssl sha512 | awk '{print $2}')
+
+    cp "$composeFilePath"/importer/volume/openhim-import-temp.json "$composeFilePath"/importer/volume/openhim-import.json
+    sed -i "s/{{PASSWORD_SALT}}/$PASSWORD_SALT/g" "$composeFilePath"/importer/volume/openhim-import.json
+    sed -i "s/{{PASSWORD_HASH}}/$PASSWORD_HASH/g" "$composeFilePath"/importer/volume/openhim-import.json
+    sed -i "s/{{OPENHIM_CLIENT_TOKEN}}/${OPENHIM_CLIENT_TOKEN-test}/g" "$composeFilePath"/importer/volume/openhim-import.json
```

Comment:
Lols is the Token `test` not prod ready 🤣 
Glad you added an easy change 😁

Reply:
:laughing: yeah probably not the most suitable for production.